webpack/assets/javascripts/charts/c3DonutChartService.js

```diff
@@ -0,0 +1,93 @@
+import * as c3 from 'c3';
+
+const enums = {
+    SIZE: {
+        SMALL: {height: 240},
```

Comment:
The size of the small chart will fit itself to the size of its container. The modal chart is still fixed size but there is no problem changing that if required.